easybuild/easyconfigs/g/graphviz/graphviz-0.20.1-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,27 @@
+easyblock = 'PythonPackage'
+
+name = 'graphviz'
```

Comment:
the name needs to be changed since this will clash with `Graphviz` in some MNS

Reply:
And we also have `pygraphviz` (which is an actual different package).

This should be fixed consistently for all `graphviz` easyconfigs though (which shouldn't be done in this PR)?

Reply:
Actually, fixed in 6fe47351b5, since the existing `graphviz` easyconfigs all use a deprecated toolchain, so there's no point in fixing those...